web/src/routes/index.svelte

```diff
@@ -4,22 +4,39 @@
 
 <script lang="ts">
   import Counter from '$lib/Counter.svelte';
+  import { tw, setup } from "twind"
+
+  setup({
```

Comment:
This will run browser-side ? Is there a way to make this run during svelte compile or on SSR/SSG ?

Reply:
I'm not sure, have to check with `npm run build` or something